web-api/src/lambdaWrapper.js

```diff
@@ -16,14 +16,26 @@ export const lambdaWrapper = lambda => {
       body: JSON.stringify(req.body),
     });
     res.status(response.statusCode);
+
+    res.set({
+      'Access-Control-Allow-Origin': '*',
```

Comment:
This doesn't look like a change in behavior, but need to be sure we are using this properly. 🤔

Reply:
We can certainly lock this down to the web client's URLs.

Reply:
@ericsorenson -- can you log an item to revisit this?

Reply:
You bet.

Reply:
https://app.zenhub.com/workspaces/flexionef-cms-5bbe4bed4b5806bc2bec65d3/issues/flexion/ef-cms/6918